machines/bleShare/scan/scanMachine.ts

```diff
@@ -701,6 +702,16 @@ export const scanMachine =
               entry: ['resetFlowType', 'resetSelectedVc'],
               always: '#scan.disconnected',
             },
+            checkFaceAuthConsentForMiniView: {
```

Comment:
Should we include Flow type (MiniView) in state name?
Asking since the logic in this state is not dependent on any flow type, given that is the case should we specify the flow type?

Reply:
Actually this state gets triggered only via mini view. Your point is correct as well. What should be done?

Reply:
@abhip2565  If the state is independent of flow type how about naming the state irrespective of flow type like "checkFaceAuthConsent" ?

Reply:
yes thats fine

Reply:
Same name states are present